### PR TITLE
Fim integration tests tools

### PR DESCRIPTION
**Pull request by vicferpoy**
This PR adds the following functions to **packages/wazuh_testing/wazuh_testing/fim.py**:

- `create_file(type, path)` : it creates a specific type of file in a given path.

- `change_internal_options(int_opt_path, pattern, value)` : it changes the value of a parameter inside `internal_options.conf`.

These are useful for testing with **FIM** using temporal files.

### CODE REVIEW

File: packages/wazuh_testing/wazuh_testing/fim.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 import json
 import os
+import socket
```
Comment (crd1985) [inline packages/wazuh_testing/wazuh_testing/fim.py]: Organize imports

File: packages/wazuh_testing/wazuh_testing/fim.py [hunk 2]
```diff
@@ -96,6 +103,62 @@ def is_fim_scan_ended():
     return -1
 
 
+def create_file(type, path):
+    """ Creates a file in a given path.
+
+    :param type: Defined constant that specifies the type. It can be: FIFO, SYSLINK, SOCKET or REGULAR
```
Comment (crd1985) [inline packages/wazuh_testing/wazuh_testing/fim.py]: Must show the data type of the param

File: packages/wazuh_testing/wazuh_testing/fim.py [hunk 3]
```diff
@@ -96,6 +103,62 @@ def is_fim_scan_ended():
...
+    :param path: Path where the file will be created
+    """
+    getattr(sys.modules[__name__], f'_create_{type}')(path)
+
+
+def _create_fifo(path):
+    fifo_path = os.path.join(path, 'fifo_file')
```
Comment (crd1985) [inline packages/wazuh_testing/wazuh_testing/fim.py]: Missing docstring

File: packages/wazuh_testing/wazuh_testing/fim.py [hunk 4]
```diff
@@ -96,6 +103,62 @@ def is_fim_scan_ended():
...
+        if os.path.exists(socket_path):
+            raise
+    sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+    sock.bind(socket_path)
+
+
+def _create_regular(path):
+    regular_path = os.path.join(path, 'regular_file')
+    with open(regular_path, 'w') as f:
+        f.write('')
```
Comment (crd1985) [inline packages/wazuh_testing/wazuh_testing/fim.py]: I miss a param to be passed here to tell the function the content of the file

File: packages/wazuh_testing/wazuh_testing/fim.py [hunk 5]
```diff
@@ -96,6 +103,62 @@ def is_fim_scan_ended():
...
+
+
+def change_internal_options(int_opt_path, pattern, value):
+    """ Changes the value of a given parameter
+
+    :param int_opt_path: internal_options.conf path
```
Comment (crd1985) [inline packages/wazuh_testing/wazuh_testing/fim.py]: Data types

File: packages/wazuh_testing/wazuh_testing/fim.py [hunk 6]
```diff
@@ -96,6 +103,62 @@ def is_fim_scan_ended():
...
+    sock.bind(socket_path)
+
+
+def _create_regular(path):
+    regular_path = os.path.join(path, 'regular_file')
+    with open(regular_path, 'w') as f:
+        f.write('')
+
+
+def change_internal_options(int_opt_path, pattern, value):
```
Comment (crd1985) [inline packages/wazuh_testing/wazuh_testing/fim.py]: This function should allow creating new params too. As a user, I would like to pass a param name and a value and the function should manage it.

File: packages/wazuh_testing/wazuh_testing/fim.py (hunk 1 above)
Comment (vicferpoy) [reply packages/wazuh_testing/wazuh_testing/fim.py]: Done

File: packages/wazuh_testing/wazuh_testing/fim.py (hunk 2 above)
Comment (vicferpoy) [reply packages/wazuh_testing/wazuh_testing/fim.py]: Done

File: packages/wazuh_testing/wazuh_testing/fim.py (hunk 3 above)
Comment (vicferpoy) [reply packages/wazuh_testing/wazuh_testing/fim.py]: Done

File: packages/wazuh_testing/wazuh_testing/fim.py (hunk 4 above)
Comment (vicferpoy) [reply packages/wazuh_testing/wazuh_testing/fim.py]: Done

File: packages/wazuh_testing/wazuh_testing/fim.py (hunk 5 above)
Comment (vicferpoy) [reply packages/wazuh_testing/wazuh_testing/fim.py]: Done

File: packages/wazuh_testing/wazuh_testing/fim.py (hunk 6 above)
Comment (vicferpoy) [reply packages/wazuh_testing/wazuh_testing/fim.py]: Done

**Pull request merged**